pages/common/ruby.md

```diff
@@ -22,3 +22,7 @@
 - Show the version of Ruby you are using:
 
 `ruby -v`
+
+- Start the built-in HTTP server on port 8080 in the current directory:
+
+`ruby -run -e httpd`
```

Comment:
```suggestion
- Start the built-in HTTP server on port 8080 in the current directory:

`ruby -run -e httpd`

- Show the version of Ruby you are using:

`ruby -v`
```

Thanks for your contribution, I have a minor suggestion. 

> We generally place the help and version examples at the last.

Reply:
I followed python's page. But whatever, no strong opinion